app/javascript/src/js/analytics.js

```diff
@@ -5,65 +5,49 @@
 // for the code doing that.
 // format: ('_trackEvent', category, action, label, value)
 
-$(() => {
+window.addEventListener('DOMContentLoaded', () => {
   // Fire a custom analytics event whenever any link is clicked
-  $('a').on('click', (e) => {
-    const $target = $(e.target);
-    let href = $target.attr('href');
-
-    // manually set href for elements wrapped in anchor tags
-    // this is because the click event target is actually the
-    // internal element, e.g. an <h3> tag wrapped in <a> tags
-    // will register a click target of <h3> not <a> so the
-    // 'href' property will be undefined and we must check its
-    // parents to acquire the href.
-    if (!href) {
-      href = $target.parents('a').attr('href');
-    }
-
-    // Send separate events for internal and external link clicks
-    const regexp = /^https?:/;
-    if (regexp.test(href)) {
-      _gaq.push(['_trackEvent', 'external_link', 'click', href, 1]);
-    } else {
-      _gaq.push(['_trackEvent', 'internal_link', 'click', href, 1]);
-    }
-  });
-
-  // Fire an event whenever a user marks a lesson completed on the lessons index page
-  // else whenever a student marks a lesson not completed on the lessons index page
-  $('[id^=section-lessons__]').on('click', function (e) {
-    if ($(this).children(':first').data('method') === 'post') {
-      _gaq.push(['_trackEvent', 'lesson_completion', 'mark_completed', $(this).children(':first')[0].href, 1]);
-    } else if (($(this).children(':first').data('method') === 'delete')) {
-      _gaq.push(['_trackEvent', 'lesson_completion', 'mark_not_completed', $(this).children(':first')[0].href, 1]);
-    }
+  document.querySelectorAll('a').forEach((a) => {
```

Comment:
Let's get rid of this tracker during this rewrite. I can't recall any instance where these internal and external event metrics were useful/needed in all the time I've been here.